uavcan/node.py

```diff
@@ -29,7 +30,6 @@
 
 logger = getLogger(__name__)
 
-
```

Comment:
This extra line was [per PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines)